packages/list-box/src/vaadin-multi-select-list-mixin.js

```diff
@@ -101,12 +101,18 @@ export const MultiSelectListMixin = (superClass) =>
         this.items.forEach((item) => {
           item.selected = false;
         });
+
+        this.removeAttribute('aria-multiselectable');
```

Comment:
I guess, in addition to snapshot tests, we need a unit test that makes sure the attribute is properly removed when `multiple` becomes `false`.

Reply:
Thanks, done.